internal/corazarules/rule_match.go

```diff
@@ -159,7 +171,7 @@ func (mr MatchedRule) AuditLog(code int) string {
 		} else {
 			log.WriteString("Coraza: Warning. ")
 		}
-		mr.matchData(matchData, log)
+		mr.matchData(log, matchData)
```

Comment:
nit, swaps parameters order just to keep consistency across `mr.matchData()` and `mr.writeDetails()`